src/DataTypes/DataTypeAggregateFunction.cpp

```diff
@@ -36,25 +38,25 @@ namespace ErrorCodes
 
 std::string DataTypeAggregateFunction::doGetName() const
 {
-    std::stringstream stream;
```

Comment:
So the std::locale destructor in Nicolae's flame graph was from stringstream? I didn't know it is so unsuitable for performance-sensitive cases.

Reply:
Yes, std::stringstream is awful for performance.